plugins/node/plugin.zsh

```diff
@@ -0,0 +1,24 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERSION=${GEOMETRY_COLOR_NODE_NPM_VERSION:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERSION=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERSION $GEOMETRY_SYMBOL_NODE_NPM_VERSION) 
+
+local node='node'
+local npm='npm'
+
+geometry_prompt_node_setup(){} 
+
+geometry_prompt_node_render(){
+	local n=$+commands[$node]
```

Comment:
Same here. No need to use a variable. Better  use:

    if (( $+commands ['node'] == 0 ))